### DISCUSSION TITLE
How to change the language of the bot's systemic messages? type: type a new question

### DISCUSSION THREAD

**Discussion by gibatzk** [microsoft/sample-app-aoai-chatGPT#1185] [General]
How can I change the language of the bot's systemic messages?

type a new question...
stop generating

**Comment by zigad** [comment]
You can create a fork and change the hardcoded messages. You can find most of the messages in **_frontend/src/components_**.

The only problem I'm facing is that I cannot translate "The requested information is not available in the retrieved data. Please try another query or topic."
